polaris-react/src/components/AppProvider/global.css

```diff
@@ -2,7 +2,7 @@
 
 /* Looking for media queries? They're imported in the postcss config in
    polaris-react/config/postcss-plugins.js */
-@import '../../../../node_modules/@shopify/polaris-tokens/dist/css/styles.css';
+@import '../../../node_modules/@shopify/polaris-tokens/dist/css/styles.css';
```

Comment:
`yarn` used to hoist everything to the root `node_modules`, but `pnpm` doesn't hoist like that, so we only need to go up to the `polaris-react/node_modules` to find the dependency.